docs/explanations/architecture/styles.md

```diff
@@ -0,0 +1,143 @@
+## Styles in the block editor
```

Comment:
This is specifically about the styles in the "content" of the block editor, not its UI which can be different right? Should we specify?

Reply:
https://github.com/WordPress/gutenberg/pull/38208/commits/f217969cf6490fc145ada824179238c9ebb951a8